test/e2e/pair-liquidity-info-history-v2-db.service.e2e-spec.ts

```diff
@@ -37,7 +37,12 @@ describe('PairLiquidityInfoHistoryV2DbService', () => {
     await prismaService.token.createMany({ data: [token1, token2, token3] });
     await prismaService.pair.createMany({ data: [pair1, pair2, pair3] });
     await prismaService.pairLiquidityInfoHistoryV2.createMany({
-      data: [historyEntry1, historyEntry2, historyEntry3, historyEntry4],
+      data: shuffle([
+        historyEntry1,
+        historyEntry2,
+        historyEntry3,
+        historyEntry4,
+      ]),
```

Comment:
why the shuffle? if there is not anything specifically we are testing I would prefer if we could test 1 or 2 fixed orders. Because if the test fails we don't really know the configuration we had and why it failed

Reply:
I wanted to test whether `getWithinHeightSortedWithPair` actually returns the entries sorted and not just in the db order. But good point, better to create a not sorted order manually.

Reply:
Done.